include/flatbuffers/flatbuffers.h

```diff
@@ -288,7 +288,7 @@ struct VectorIterator : public
   }
 
   VectorIterator operator++(int) {
-    VectorIterator temp(data_);
+    VectorIterator temp(data_,0);
```

Comment:
I suppose, it will be good for the prefix increment, because postfix increment should return old value of object and olny then incremented. Correct me, if I am wrong.